services/horizon/internal/test/integration/integration.go

```diff
@@ -42,14 +44,41 @@ type Config struct {
 	ProtocolVersion       int32
 	SkipContainerCreation bool
 	CoreDockerImage       string
+
+	// Weird naming here because bools default to false, but we want to start
+	// Horizon by default.
+	DontStartHorizon bool
```

Comment:
Nit/Suggestion:
```suggestion
	SkipHorizon bool
```
Above we called it `SkipContainerCreation`. Maybe there's a similar phrasing here? `SkipHorizonStart`?

Reply:
yep, hated my naming; great suggestion